src/convertToObject.js

```diff
@@ -1,18 +1,22 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const formatedStyles = {};
+
+  let propertyArray = sourceString.replace(/[\n]/gi, '').split(';');
+
+  propertyArray = propertyArray.filter(element => element.length > 2);
+
+  propertyArray.map((element) => {
+    const key = element.slice(0, element.indexOf(':')).trim();
+    const value = element
+      .slice(element.indexOf(':') + 1, element.length)
```

Comment:
you look for `indexOf(':')` 2 times, just split string by that symbol, it'll be easier